src/Tasks/AlgoliaIndexTask.php

```diff
@@ -15,13 +15,18 @@
 
 /**
  * Class AlgoliaIndexTask
+ *
+ * This task will connect with your algolia environment based on the provided configuration and sync Pages to algolia.
+ * The task create algolia objects containing data and provides also a solution to sync localised data.
+ * For more information about the task see the README.MD
+ *
  * @package AlgoliaSyncModuleDirectLease
  */
 class AlgoliaIndexTask extends BuildTask
 {
 
     protected $title = 'DirectLease AlgoliaIndexTask';
-    protected $description = "This task will synchronize all Pages with ShowSearch on true with algolia. To perform a full wipe/insert use param fullsync=1";
+    protected $description = "This task will synchronize all published Pages with the Page value ShowInSearch(see CMS->Page->Settings) on true to Algolia";
```

Comment:
Wait, what? Ik dacht dat dat ShowInSearch deel niet klopt... Maar je gebruikt de ingebouwde "ShowInSearch" van Silverstripe om te bepalen of een page wel/niet wordt gesynced naar Algolia? Dan moet dat ook ff in de readme duidelijk worden :-D